x-pack/plugins/security_solution/public/management/common/constants.ts

```diff
@@ -10,7 +10,7 @@ import { SecurityPageName } from '../../app/types';
 // --[ ROUTING ]---------------------------------------------------------------------------
 export const MANAGEMENT_APP_ID = `${APP_ID}:${SecurityPageName.administration}`;
 export const MANAGEMENT_ROUTING_ROOT_PATH = '';
-export const MANAGEMENT_ROUTING_HOSTS_PATH = `${MANAGEMENT_ROUTING_ROOT_PATH}/:tabName(${AdministrationSubTab.hosts})`;
+export const MANAGEMENT_ROUTING_ENDPOINTS_PATH = `${MANAGEMENT_ROUTING_ROOT_PATH}/:tabName(${AdministrationSubTab.endpoints})`;
```

Comment:
Just so that we're all in sync - this is a "breaking" change for 7.10. The previous URL (`/hosts`) will no longer be valid.
A way around this is to provide a `<Redirect>` from old -> new route. I think we might be able to do that in (`/management/routes.tsx`). (maybe track/implement in separate ticket)

Reply:
we probably should also change the `MANAGEMENT_STORE_HOSTS_NAMESPACE` below on line 25. its the property in Redux store that will hold the Endpoints list State.

Reply:
> Just so that we're all in sync - this is a "breaking" change for 7.10. The previous URL (`/hosts`) will no longer be valid.
A way around this is to provide a `<Redirect>` from old -> new route. I think we might be able to do that in (`/management/routes.tsx`). (maybe track/implement in separate ticket)

Good point, I think this is a good solution to support any saved links from `7.9`.  I agree that we can do this in another ticket - created one already for tracking, will link in the original

Reply:
@paul-tavares I think I got the namespace change already? Check and tell me if that's not what you meant

Reply:
@kevinlog its something we will need to discuss/mention with the entire team. Now that we are going Beta, should we account for transition/Migration in anything we do going forward?  If we don't have to worry about just yet (because we're Beta now), then maybe we don't have to do it.

Reply:
@pzl yeah, I see that now. Thanks.